src/app/main.cpp

```diff
@@ -187,6 +189,19 @@ int main(int argc, char *argv[])
         std::cerr << "Couldn't set environment variable...\n";
 
 #ifndef DISABLE_GUI
+
+    // Set dark theme
+    if (params.darkTheme) {
+        QFile file(":/qdarkstylesheet.qss");
```

Comment:
Wouldn't it be more useful to put the stylesheet outside of the application binary to allow for possible user edits?